3DAmsterdam/Assets/Amsterdam3D/Scripts/Sewer/SewerageGenerator.cs

```diff
@@ -80,6 +80,7 @@ private void ClearNetwork()
             //Clears the network of spawned prefabs, so we can load a new area
             foreach(Transform child in networkContainer)
             {
+                DestroyImmediate(child.GetComponentInChildren<MeshFilter>().mesh);
```

Comment:
moest dit niet sharedMesh zijn? -,anders bleef er alsnog wat hangen meende ik

Reply:
aangepast in nieuwe commit